src/website/views/custom-elements/button.pug

```diff
@@ -116,16 +116,16 @@ p.-text
   .-p--3.-d--flex
     chi-button(type='icon')
       chi-icon(icon='atom')
-    chi-button(flat, class='-ml--2', type='icon')
+    chi-button(variant='flat', class='-ml--2', type='icon')
```

Comment:
```suggestion
    chi-button.-ml--2(variant='flat, type='icon')
```
@srinath-koneru This is the preferred way to include classes in Pug. Use the `class=` notation when using variables.

Reply:
Alright.